index.js

```diff
@@ -17,8 +17,13 @@ const onClosed = () => {
 
 const createMainWindow = () => {
     const win = new electron.BrowserWindow({
-        width  : 1000,
-        height : 700
+        width          : 600,
+        height         : 400,
+        maxHeight      : 700,
+        maxWidth       : 800,
+        minHeight      : 400,
+        minWidth       : 500,
+        fullscreenable : false
```

Comment:
Actually, I don't think we should impose any maximums or disallow full screen.

Reply:
Ok. Got you.